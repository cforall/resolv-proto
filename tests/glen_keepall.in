// based on example on p.93 of Glen's thesis
//
// Expected output:
// f( g-l( 1 => 2 ) => 3 )
f 3
4 g-i 1
2 g-l 2
%%
f( g( 1 ) )
