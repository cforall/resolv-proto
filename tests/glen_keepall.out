
[Void / (0,0,0,0,2)] f( g-l( 1 => 2 ) => 3 )
